lamindb_setup/core/_hub_core.py

```diff
@@ -251,7 +254,13 @@ def _delete_instance(
             check_storage_is_empty(
                 root_path, account_for_sqlite_file=account_for_sqlite_file
             )
-    _update_instance_record(instance_with_storage["id"], {"storage_id": None}, client)
+    # remove this code block after migrtion, i.e. deleting 'storage_id' column of 'instance'
```

Comment:
We can also remove it straight away if we just set the values to `None` in the hub. I'd actually prefer that.

Reply:
I would prefer to keep it until we actually migrate.

Reply:
Why? I don't see the reason.

It makes the code complicated and unreadable. There is a really nice potential for a refactor once we remove this because the whole backward forward coupling of instance and storage is then finally disentangled.

Reply:
Because we're no longer populating this column the tests are already passing even without deleting anything on the hub.

We still have to go through with it though so that deletions of legacy instances don't start failing.

Reply:
Made an issue for this: https://github.com/laminlabs/laminhub/issues/1748